keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -452,94 +462,200 @@ change are understandable. This may include API specs (though not always
+
 ![Privileged Container Diagram](Privileged.png)
 
 #### Networking
 - The container will be in the host’s network namespace (default network compartment) so it will have access to all the host’s network interfaces and have the host's IP as well.
+
 #### Resource Limits
 - Resource limits (disk, memory, cpu count) will be applied to the job and will be job wide. For example, with a limit of 10 MB is set for the job, if every process in the jobs memory allocations added up exceeds 10 MB this limit would be reached. This is the same behavior as other Windows container types. These limits would be specified the same way they are currently for whatever orchestrator/runtime is being used.
+
 #### Container Lifecycle 
 - The container's lifecycle will be managed by the container runtime just like other Windows container types.
-#### Container users 
--  The privileged container should be able to run as any user that's available on the host or in the domain of the host machine. Password accounts are being investigated.
+
+#### Container users
+
+- The privileged container can run as any user that's available on the host or in the domain of the host machine. Password accounts are being investigated.
+Running privileged containers as non SYSTEM/admin accounts will be the primary way operators can restrict access to system resources (files, registry, named pipes, WMI, etc).
+More information on Windows resource access can be found at https://docs.microsoft.com/en-us/archive/msdn-magazine/2008/november/access-control-understanding-windows-file-and-registry-permissions.
+
 #### Container Mounts
-- Directory mounts (i.e. directory C:\test mapped to C:\test in the privileged container) are still being investigated. Note that file system mapping in the current implementation would expose both the host filesystem and the container filesystem to the processes in the privileged container. We are still investigation how external mounts such as secrets or storage accounts would be added. The default visibility of all the host filesystem may present backwards compatibility challenges in the future if host path mounting is enabled in Windows. Currently the only method to restrict would be via the user in a way similar to [UID or GID](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) in the PSP. Currently however, this host path mounting is unlikely to be enabled in the future as a restriction of Job Objects.  
+
+- When privileged containers are started a new volume will be created on the host which will contain the contains of the container image. Containers will have a default working directory that points to this container volume. Containers will also have full access to the host file-system (unless restricted by filed-based ACLs and the run_as_username used to start the container.) Processes should use absolute paths when accessing files on the host and relative paths when accessing files brought in via the container image.
```

Comment:
@ddebroy @jingxu97 @msau42 can you help review this for sig-storage per feedback from @liggitt?

Reply:
nit: which will contain the contains of the container image?

Reply:
do you have an example about absolute path and relative paths in container/pod spec?

Reply:
The absolute paths vs relative paths discussed here are guidelines for how programs running in the containers can access either host resources or resources in volume months. No changes are planned for how volumes are expressed in deployment specs.